requirements.txt

```diff
@@ -1 +1 @@
-Flask==2.1.1
+Flask
```

Comment:
it's not a good practice to install a dependency without a version specified

Reply:
i will edit it